spec/requests/stories_index_spec.rb

```diff
@@ -171,6 +181,10 @@
   end
 
   describe "GET podcast index" do
+    include_examples "redirects to the lowercase route" do
+      let(:path) { "/#{create(:podcast).slug.upcase}" }
```

Comment:
small suggestion, if you only need the slug, you don't need to create the object in the database, using `create`, but you can simply build it in memory, using `build()` (same parameters).

For example:

```ruby
irb(main):004:0> FactoryBot.build(:podcast).slug.upcase
=> "SLUG-3"
```

same for user and organization :-)

Reply:
Good suggestion; I'll make this change!